docs/source/data_conversion/data_conversion.rst

```diff
@@ -0,0 +1,278 @@
+
+- Serial Output: list of dictionaries, where the number of events in the file :code:`n_events` satisfies :code:`len(output) = n_events`. I.e each element in the list is a dictionary, and each field in the dictionary is the output of a single extractor. If this is provided, the :code:`DataConverter` will automatically assign event ids.
+
+- Vectorized Output: A single dictionary where each field contains a single dataframe, which holds the data from the :code:`Extractor` for the entire file. In this case, the :code:`Reader` must itself assign event ids. This method is faster if your files are not storing events serially. 
+
+In addition, classes inheriting from :code:`GraphNeTFileReader` must set class properties :code:`accepted_file_extensions` and :code:`accepted_extractors`.
+
+.. raw:: html
+
+    <details>
+    <summary><b>Example of a Reader</b></summary>
+    
+Implementing a :code:`GraphNeTFileReader` to read data from your experiment requires writing just a few lines of code. 
+Below is an example of a reader meant to parse and extract data from :code:`.h5` files from LiquidO, which output data in the vectorized format.
+
+.. code-block:: python
+
+    from typing import List, Union, Dict
+    from glob import glob
+    import os
+    import pandas as pd
+
+    from graphnet.data.extractors.liquido import H5Extractor
+    from .graphnet_file_reader import GraphNeTFileReader
+
+    class LiquidOReader(GraphNeTFileReader):
+        """A class for reading h5 files from LiquidO."""
+
+        _accepted_file_extensions = [".h5"]
+        _accepted_extractors = [H5Extractor]
+
+        def __call__(self, file_path: str) -> Dict[str, pd.DataFrame]:
+            """Extract data from single parquet file.
+
+            Args:
+                file_path: Path to h5 file.
+
+            Returns:
+                Extracted data.
+            """
+            # Open file
+            outputs = {}
+            for extractor in self._extractors:
+                output = extractor(file_path)
+                if output is not None:
+                    outputs[extractor._extractor_name] = output
+            return outputs
+
+        def find_files(self, path: Union[str, List[str]]) -> List[str]:
+            """Search folder(s) for h5 files.
+
+            Args:
+                path: directory to search for h5 files.
+
+            Returns:
+                List of h5 files in the folders.
+            """
+            files = []
+            if isinstance(path, str):
+                path = [path]
+            for p in path:
+                files.extend(glob(os.path.join(p, "*.h5")))
+            return files
+
+
+.. raw:: html
+
+    </details>
+
+
+:code:`Extractors`
+~~~~~~~~~~~~~~~~~
+
+Because rarely all the data available in files from experiments are needed for training deep learning models, GraphNeT uses :code:`Extractors` to extract only 
```

Comment:
> Because rarely all the data available in files from experiments are needed for training deep learning models, GraphNeT uses :code:`Extractors` to extract only... 

⬇️ 
Rarely _all_ the data ... deep learning models, therefore Graphnet uses ...